test/integration_test.go

```diff
@@ -36,7 +36,7 @@ func (suite *IntegrationTestSuite) cleanUp() {
 func (suite *IntegrationTestSuite) SetupSuite() {
 	suite.cleanUp()
 
-	cfg, err := config.Parse("../misc/config/config.yaml.tmpl")
+	cfg, err := config.Parse("../misc/config/config.yaml.nydus.tmpl")
```

Comment:
Could you add `estargz` tests to make sure it passes CI?

Reply:
ok, looks like we don't have CI yet. b___b

Reply:
Will add an integration test later.